src/main/java/frc/robot/Constants.java

```diff
@@ -69,7 +73,9 @@ public static class TurretConstants {
     public static final int TURRET_MOTOR_ID = 9;
     public static final int TURRET_CANcoder_ID = 17; // replace with actual CANcoder ID
     public static final int PITCH_MOTOR_ID = 10;
-    public static final int PITCH_CANcoder_ID = 18; // replace with actual CANcoder ID
+    public static final int PITCH_ENCODER_ID_1 = 1; // replace with actual ID
```

Comment:
What are these? Why are there two of them? Could they be given more descriptive names than 1 and 2?